website/docs/language/expressions/dynamic-blocks.html.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: "language"
 page_title: "Dynamic Blocks - Configuration Language"
-description: "`dynamic` blocks automatically construct multi-level, nested block structures. Learn to configure `dynamic` blocks and understand their behavior." 
+description: "Dynamic blocks automatically construct multi-level, nested block structures. Learn to configure `dynamic` blocks and understand their behavior." 
```

Comment:
Found some backtics!

Reply:
ahhhh thank you!!! This is what I get for trying to go fast :p